### PR TITLE
show courrse of user

### PR DESCRIPTION
**Pull request by VanHau95**
image

### CODE REVIEW

**Comment by VanHau95** [comment]
ready

**Comment by VanHau95** [comment]
ready

**Comment by VanHau95** [comment]
ready

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,15 +1,46 @@
...
+          </div>
+          <% end %>
+        </div>
+      </div>
+    </div>
+    <div class="clearfix"> </div>
+  </div>
 </div>
+
+
```
Comment (tranducquoc) [inline app/views/users/show.html.erb]: remove 2 dòng này đi

**Comment by tranducquoc** [comment]
commented

**Comment by VanHau95** [comment]
ready

**Comment by VanHau95** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**